util-logging/src/main/scala/com/twitter/logging/FileHandler.scala

```diff
@@ -166,8 +166,10 @@ class FileHandler(
   }
 
   private def openStream(): OutputStream = {
-    val dir = new File(filename).getParentFile
+    val file = new File(filename)
+    val dir = file.getParentFile
     if ((dir ne null) && !dir.exists) dir.mkdirs
+    bytesWrittenToFile = if (file.exists()) file.length() else 0
```

Comment:
It's a bit awkward this is setting ```bytesWrittenToFile``` outside an immediate synchronized block which seems like it would be safer. It does appear it's wrapped in a synchronized block by callers though.